source/Calamari.Tests/KubernetesFixtures/Approved/KubernetesContextScriptWrapperFixture.ExecutionWithEKS_IAMAuthenticator.approved.txt

```diff
@@ -6,5 +6,12 @@
 [Verbose] "kubectl" config use-context octocontext --request-timeout=1m
 [Verbose] "kubectl" config set-cluster octocluster --insecure-skip-tls-verify=true --request-timeout=1m
 [Info] Creating kubectl context to <server> (namespace calamari-testing) using EKS cluster name my-eks-cluster
+[Error] The EKS cluster Url specified should contain a valid aws region name
+[Verbose] Unable to authenticate to <server> using the aws cli. Failed with the error: System.ArgumentNullException: Value cannot be null. (Parameter 's')
+   at System.IO.StringReader..ctor(String s)
+   at Newtonsoft.Json.Linq.JObject.Parse(String json, JsonLoadSettings settings)
+   at Newtonsoft.Json.Linq.JObject.Parse(String json)
+   at Calamari.Kubernetes.KubernetesContextScriptWrapper.SetupKubectlAuthentication.SetupContextForAmazonServiceAccount(String namespace, String clusterUrl, String user) in C:\Users\isaac\source\repos\Calamari\source\Calamari\Kubernetes\KubernetesContextScriptWrapper.cs:line 456.

```

Comment:
Note: I think throwing an error here to notify why the aws-cli auth failed is the right thing but the fact the error includes the line number etc is a real pain for these approval tests.